internal/deviations/deviations.go

```diff
@@ -67,6 +67,12 @@ import "flag"
 
 // Vendor deviation flags.
 var (
+	PortSpeedDuplexAutoNegotiateEnabled = flag.Bool("port_speed_enabled", false,
```

Comment:
Please explain what is the device behavior that needs to be deviated, and the expected OC behavior so we know when the device becomes OC compliant, then this deviation can be removed.

Look at other deviations to see how they are worded. We cannot accept deviations without justification.

Reply:
@liulk liulk , Addressed review commenst